<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditItemDialog</class>
 <widget class="QDialog" name="EditItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>333</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLineEdit" name="mailEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="roleLabel">
       <property name="text">
        <string>Role</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="forenameLabel">
       <property name="text">
        <string>Forename</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="phoneEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="companyLabel">
       <property name="text">
        <string>Company</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="companyEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="forenameEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="roleEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="surnameEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>Phone</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="surnameLabel">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mailLabel">
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>surnameEdit</tabstop>
  <tabstop>forenameEdit</tabstop>
  <tabstop>companyEdit</tabstop>
  <tabstop>roleEdit</tabstop>
  <tabstop>mailEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
